gladier_tools/posix/tar.py

```diff
@@ -1,22 +1,20 @@
-from gladier import GladierBaseTool, generate_flow_definition
+from gladier import GladierBaseTool
 
 
 def tar(**data):
     import os
     import tarfile
     import pathlib
-    tar_input = data['tar_input']
-    
-    if '~' in tar_input:
-        tar_input = os.path.expanduser(tar_input)
-        
-    path = pathlib.PurePath(tar_input)
-    os.chdir(path.parent)
 
+    tar_input = pathlib.Path(data['tar_input']).expanduser()
```

Comment:
We should agree to use pathlib as standard going forward.

Reply:
I prefer it over the os.path module. It's both easier to use, cleaner to read, and results in less bugs.